site/content/extend/plugins/webapp/best-practices.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Design Best Practices
+date: 2019-06-02T00:00:00-00:00
+subsection: Web App Plugins
+weight: 0
+---
+
+
+### Actions that apply to specific Channels
+- Recommendation: Have your plugin register the actions to [the channel header](https://developers.mattermost.com/extend/plugins/webapp/reference/#registerChannelHeaderButtonAction). This makes it quickly accessible by users and actions apply to the channel they're viewing. 
+- Example:  Zoom meeting posts to a channel
+
+![Custom Channel Header Button](/img/extend/bp-channel-header.png)
```

Comment:
Looks like the images aren't rendering on the preview

Reply:
I don't think it renders unless you have the server running on your machine pointing at the repo, right?